src/sentry/static/sentry/app/utils/withPlugins.tsx

```diff
@@ -83,8 +83,8 @@ const withPlugins = <P extends InjectedPluginProps>(
         render() {
           return (
             <WrappedComponent
-              {...(this.props as P)}
-              plugins={this.state.store as Plugin[]}
+              {...(this.props as P & WithPluginProps)}
+              plugins={this.state.store}
```

Comment:
Does this get typed correctly?

Reply:
@billyvg well, right now the store is typed as `any`. But when I convert the store file I'm hopeful it will be correct. Either way, the original was wrong as I mentioned in the other comment.